src/test/librbd/journal/test_Entries.cc

```diff
@@ -172,6 +172,12 @@ TEST_F(TestJournalEntries, AioDiscard) {
   ASSERT_EQ(0, c->wait_for_complete());
   c->put();
 
+  CephContext* cct = reinterpret_cast<CephContext*>(_rados.cct());
+  if (cct->_conf->rbd_skip_partial_discard) {
+    ASSERT_FALSE(wait_for_entries_available(ictx));
```

Comment:
I did not see journal entry being created and I thought it was ok.

I suppose for journal entry to be created, `associate_journal_event()` should be called, which is in `AioImageDiscard::append_journal_event()`, and this only happens when sending object extents?